src/aglais/Consumer_.h

```diff
@@ -0,0 +1,50 @@
+/* -*- mode: c++ -*-
```

Comment:
Not a big fan of filenames with trailing underscores. I understand why it's there, but perhaps we could find a name that doesn't need one?